### DISCUSSION TITLE
Ability to duplicate entries in Custom Source Views (CraftCMS 4)

### DISCUSSION THREAD

**Discussion by adrianjean** [craftcms/cms#12382] [Ideas] [answered]
In the CP, under Entries, in a "normal" listing of entries, you can select some entries and use the Cog/Gear menu to duplicate those entries. 

However in custom Source Views, you don't have that option — only the ability to delete, view, edit.

Would be great to have the same entry actions in the CP so that I can use Source Views to better organize content.
(Ignore the "Bulk Edit" in the screenshot)

<img width="856" alt="Screenshot 2022-11-25 at 5 18 02 PM" src="https://user-images.githubusercontent.com/3161182/204060090-8f2c7f36-7cb6-4e88-bf83-0d6183548130.png">

**Comment by brandonkelly** [comment, marked as answer]
This has been added for Craft 4.4 via 466849e9aeb8549b02b7b447d8368ecafc7b996b 🎉

**Reply by adrianjean** [reply]
You guys are amazing. Thank you!

**Reply by brandonkelly** [reply]
Craft 4.4 is out with that change!
